controllers/incomeController.js

```diff
@@ -1,5 +1,6 @@
 const Income = require('../models/Income');
 const incomeSchema = require('../models/incomeJoi');
```

Comment:
zaciągasz tego { checkAuth } i nigdzie go nie używasz - to powinno być w routach